compositor_api/Cargo.toml

```diff
@@ -9,9 +9,12 @@ decklink = ["compositor_pipeline/decklink"]
 web_renderer = ["compositor_render/web_renderer"]
 
 [dependencies]
-compositor_pipeline = { workspace = true }
 compositor_render = { workspace = true }
 serde = { workspace = true }
 schemars = { workspace = true }
 bytes = { workspace = true }
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+compositor_pipeline = { workspace = true }
```

Comment:
it would be cleaner to control that with a feature, but if it's fine if you want to leave it this way